docs/guides/admin/docs/upgrade.md

```diff
@@ -66,6 +66,8 @@ Configuration Changes
   option.
 - `etc/org.opencastproject.scheduler.impl.SchedulerServiceImpl.cfg` has a new option `maintenance` which temporarily
   disables the scheduler if set to `true`.
+- `etc/org.opencastproject.organization-mh_default_org.cfg` the variable `{{id}}` 
+  from `prop.player` has changed to `#{id}`.
```

Comment:
As mentioned already, please remove this. Looking at releases, the `{{id}}` syntax never existed in **any** release. Hence, no update is required. However, there is the matter of upgrading from the old, hard-coded URLs for which this file does contain an upgrade section which needs to be adapted.

Reply:
But what about the people who use the packages from version 7.0 if they upgrade to an upcoming version 7.1?

Reply:
If I merge this now, the 7.0 package already has the format `#{id}`. No version ever used `{{id}}`.

Reply:
Of course! You have right!, Ok I will delete that, even I am the release manager and we haven't release this version 😋!!

I will delete that line right now.